resources/install.rb

```diff
@@ -124,11 +136,25 @@
       source download
     end
 
-    archive_file "#{package_name}.zip" do
-      path zipfile
-      destination "#{Chef::Config[:file_cache_path]}/habitat"
-      overwrite true
-      action :extract
+    if Chef::VERSION < 15
+      ruby_block 'name' do
+        block do
+          require 'zip'
+          Zip::File.open(zipfile) do |zip_file|
```

Comment:
should we have a not_if on this ruby_block as well?

Reply:
This one didn't have a guard because it is in the upgrade action, for upgrade we want it to run no matter what